src/services/apiService.js

```diff
@@ -71,3 +71,16 @@ export const startQuizSession = (quizId) =>
       toast.error('An error occurred while starting the quiz');
       console.error(`Error starting quiz: ${e}`);
     });
+
+export const fetchResults = async (code) => {
+  try {
+    const response = await axios.get(`https://api.edukona.com/quiz-session-results/${code}`, {
+      headers: {
+        Authorization: `Token ${token}`,
+      },
+    });
+    return response.data.results;
+  } catch (error) {
+    console.error('Failed to fetch quiz results:', error);
+  }
+};
```

Comment:
For consistency, we should use `api.get`, similar to the other functions in this file

and this: 
`setResults(fetchResults(code));` 
    can become 
`fetchResults(code).then((res) => setResults(res));`